CHANGELOG.md

```diff
@@ -1,5 +1,117 @@
+  * Added iscsi volume plugin (#5506)
+  * Added glusterfs volume plugin (#6174)
+  * AWS EBS volume support (#5138)
+* Updated to heapster version to v0.10.0 (#6331)
+* Updated to etcd 2.0.9 (#6544)
+* Updated to Kibana to v1.2 (#6426)
+* Bug Fixes
+  * Kube-proxy now updates iptables rules if a service's public IPs change (#6123)
+  * Retry kube-addons creation if the initial creation fails (#6200)
+  * Make kube-proxy more resiliant to running out of file descriptors (#6727)
+
+## 0.14.2
+ * Fix a regression in service port handling validation
+ * Add a work around for etcd bugs in watch
+
+## 0.14.1
+ * Fixed an issue where containers with hostPort would sometimes go pending forever. (#6110)
+
+## 0.14.0
+ * Add HostNetworking container option to the API.
+ * PersistentVolume API
+ * NFS volume plugin fixed/re-added
+ * Upgraded to etcd 2.0.5 on Salt configs
+ * .kubeconfig changes
+ * Kubelet now posts pod status to master, versus master polling.
+ * All cluster add-on images are pulled from gcr.io
+
+## 0.13.2
+ * Fixes possible cluster bring-up flakiness on GCE/Salt based clusters
+ 
+
+## 0.12.2
+ * #5348 - Health check the docker socket and Docker generally
+ * #5395 - Garbage collect unknown containers
+
+## 0.12.1
+ * DockerCache doesn't get containers at startup (#5115)
+ * Update version of kube2sky to 1.1 (#5127)
+ * Monit health check kubelet and restart unhealthy one (#5120)
+
+## 0.12.0
+ * Hide the infrastructure pod from users
+ * Configure scheduler via JSON
+ * Improved object validation
+ * Improved messages on scheduler failure
+ * Improved messages on port conflicts
+ * Move to thread-per-pod in the kubelet
+ * Misc. kubectl improvements
+ * Update etcd used by SkyDNS to 2.0.3
+ * Fixes to GCE PD support
+ * Improved support for secrets in the API
+ * Improved OOM behavior
+
+## 0.11
+* Secret API Resources
+* Better error handling in various places
+* Improved RackSpace support
+* Fix ```kubectl``` patch behavior
+* Health check failures fire events
+* Don't delete the pod infrastructure container on health check failures
+* Improvements to Pod Status detection and reporting
+* Reduce the size of scheduled pods in etcd
+* Fix some bugs in namespace clashing
+* More detailed info on failed image pulls
+* Remove pods from a failed node
+* Safe format and mount of GCE PDs
+* Make events more resilient to etcd watch failures
+* Upgrade to container-vm 01-29-2015
+
+## 0.10
+   * Improvements to swagger API documentation.
+   * Upgrade container VM to 20150129
+   * Start to move e2e tests to Ginkgo
+   * Fix apiserver proxy path rewriting
+   * Upgrade to etcd 2.0.0
+   * Add a wordpress/mysql example
+   * Improve responsiveness of the master when creating new pods
+   * Improve api object validation in numerous small ways
+   * Add support for IPC namespaces
+   * Improve GCE PD support
+   * Make replica controllers with node selectors work correctly
+   * Lots of improvements to e2e tests (more to come)
+
```

Comment:
The markdown renders 100% exactly the same.  Do you really want me to do this?  (I copy/pasted from the release notes page)